webapp/components/CommentList/CommentList.spec.js

```diff
@@ -1,10 +1,13 @@
 import { config, mount } from '@vue/test-utils'
 import CommentList from './CommentList'
+import Comment from '~/components/Comment/Comment'
 import Vuex from 'vuex'
+import Vue from 'vue'
 
 const localVue = global.localVue
 
 localVue.filter('truncate', string => string)
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-22T15:11:15Z" title="Wednesday, January 22nd 2020, 4:11:15 pm +01:00">Jan 22, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -97,5 +97,22 @@ describe('CommentList.vue', () => {
         })
       })
     })
+
+    describe('Comment', () => {
+      beforeEach(() => {
+        wrapper = Wrapper()
+      })
+      it.only('Comment emitted reply()', () => {
```


```suggestion
      it('Comment emitted reply()', () => {
```
Remove `.only` …